examples/custom_test_runner/MODULE.bazel

```diff
@@ -0,0 +1,56 @@
+module(
+    name = "custom_test_runner_example",
```

Comment:
Note that unpublished modules don't need a `module` declaration, if you want to leave it out for simplicity.

Reply:
Thanks. I did it to give the workspace a name like I do in the `WORKSPACE` file.